src/main/java/org/fundacionjala/at15/pokemon/commands/create/CreateWildPokemonBattle.java

```diff
@@ -18,12 +19,22 @@
     @Option(names = { "-btlpokeID", "-bp" }, description = "Wild pokemon ID")
     private String pokemonID;
 
+    @Option(names = { "--xml" }, description = "create the object in xml format")
+    private boolean xml;
+
     @Override
     public Integer call() {
         Pokemon poke = (Pokemon) Reader.read(pokemonID);
         Trainer trainer = (Trainer) Reader.read(trainerID);
         WildPokemonBattle wildBattle = new WildPokemonBattle(trainer, poke);
-        wildBattle.write();
+
+        if (!xml) {
```

Comment:
I think Wild Pokemon Battle is not implemented for the reader. I mention it just in case

Reply:
👍🏼 There should be an exception!